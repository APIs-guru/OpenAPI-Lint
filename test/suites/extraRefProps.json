{
  "title": "extra reference properties",
  "patch": {
    "paths": {
      "/pet": {
        "put": {
          "parameters": {
            "0": {
              "schema": {
                "extraField": "test"
              }
            }
          }
        }
      }
    }
  }
}
