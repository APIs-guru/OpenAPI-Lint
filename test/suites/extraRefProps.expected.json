{
  "EXTRA_REFERENCE_PROPERTIES": [
    {
      "code": "EXTRA_REFERENCE_PROPERTIES",
      "params": [
        "extraField"
      ],
      "type": "warning",
      "message": "Extra JSON Reference properties will be ignored: extraField",
      "path": [
        "paths",
        "/pet",
        "put",
        "parameters",
        "0",
        "schema"
      ]
    }
  ]
}